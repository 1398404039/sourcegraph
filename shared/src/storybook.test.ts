import initStoryshots from '@storybook/addon-storyshots'
import * as path from 'path'

initStoryshots({ framework: 'react', configPath: path.resolve(__dirname, '..', '.storybook') })
