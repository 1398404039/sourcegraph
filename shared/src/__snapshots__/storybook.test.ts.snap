// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Toggle disabled/off 1`] = `
<button
  className="toggle toggle__disabled "
  onClick={[Function]}
  value={0}
>
  <span
    className="toggle__bar  toggle__bar--disabled"
  />
  <span
    className="toggle__knob  toggle__knob--disabled"
  />
</button>
`;

exports[`Storyshots Toggle disabled/on 1`] = `
<button
  className="toggle toggle__disabled "
  onClick={[Function]}
  value={1}
>
  <span
    className="toggle__bar toggle__bar--active toggle__bar--disabled"
  />
  <span
    className="toggle__knob toggle__knob--active toggle__knob--disabled"
  />
</button>
`;

exports[`Storyshots Toggle interactive 1`] = `
<div
  className="d-flex align-items-center"
>
  <button
    className="toggle  mr-2"
    onClick={[Function]}
    title="Hello"
    value={0}
  >
    <span
      className="toggle__bar  "
    />
    <span
      className="toggle__knob  "
    />
  </button>
   Value is
   
  undefined
</div>
`;

exports[`Storyshots Toggle off 1`] = `
<button
  className="toggle  "
  onClick={[Function]}
  value={0}
>
  <span
    className="toggle__bar  "
  />
  <span
    className="toggle__knob  "
  />
</button>
`;

exports[`Storyshots Toggle on 1`] = `
<button
  className="toggle  "
  onClick={[Function]}
  value={1}
>
  <span
    className="toggle__bar toggle__bar--active "
  />
  <span
    className="toggle__knob toggle__knob--active "
  />
</button>
`;
